packages/atomic/src/components/atomic-folded-result-list/atomic-folded-result-list.tsx

```diff
@@ -107,11 +88,17 @@ export class AtomicFoldedResultList implements InitializableComponent {
    */
   @Prop({reflect: true}) imageSize?: ResultDisplayImageSize;
```

Comment:
`imageSize` should no longer be optional. It was optional in `atomic-result-list` because `image` was deprecated.

Reply:
aah ok